### PR TITLE
Jjb macros

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: jjb/mkck/mkck-trigger.yaml [hunk 1]
```diff
@@ -508,7 +524,9 @@
                 - snippets/mkck-target-properties.bash
         - inject:
             properties-file: 'target.properties'
-        - shell: "git clone https://github.com/suse/sesci -b $CI_BRANCH"
+        - git-clone:
+            git_url: "https://github.com/suse/sesci"
+            git_branch: "$CI_BRANCH"
```
Comment (kshtsk) [inline jjb/mkck/mkck-trigger.yaml]: Looks good, but I'm not sure that using macro for one-liner is efficient, especially :-) if to replace it with more lines of code

File: jjb/mkck/mkck-trigger.yaml [hunk 2]
```diff
@@ -180,16 +222,9 @@
...
-            virtualenv v
-            . v/bin/activate
-            pip install -U pip
-            pip install python-openstackclient openstacksdk docopt PyYaml paramiko
-        - shell: |
-            . v/bin/activate
-            STATUS=$PWD/mkck.status
-            cd sesci && OS_CLOUD=$TARGET_CLOUD python -u os-server.py -a create -f conf/$CONF.yaml -s $STATUS
+        - git-clone-sesci-suse-master
+        - create-openstackclient-venv
```
Comment (kshtsk) [inline jjb/mkck/mkck-trigger.yaml]: That's not the openstackclient-venv, I would say it is 'sesci' env

File: jjb/mkck/mkck-trigger.yaml (hunk 1 above)
Comment (toabctl) [reply jjb/mkck/mkck-trigger.yaml]: True. do you want me to remove that commit?

File: jjb/mkck/mkck-trigger.yaml (hunk 2 above)
Comment (toabctl) [reply jjb/mkck/mkck-trigger.yaml]: done

File: jjb/mkck/mkck-trigger.yaml (hunk 1 above)
Comment (kshtsk) [reply jjb/mkck/mkck-trigger.yaml]: yes, please

Comment (toabctl) [reply jjb/mkck/mkck-trigger.yaml]: @kshtsk done

**Review by kshtsk — APPROVED** [review]
lgtm, I'm ok to merge as soon as you tried to populate this someone and it is working

**Comment by toabctl** [comment]
@kshtsk I updated the `jjb/mkck/mkck-trigger.yaml` jobs now. Let's see if it fails somewhere...

**Pull request merged**